### PR TITLE
docs: add guide for SSO via Google

### PR DESCRIPTION
**Pull request by anuveyatsu**
## Related issue

@aeneasr there is no issue I believe for this specific item but a guide for Google should be good to have.

## Proposed changes

This PR adds instructions on how to setup SSO via Google.

## Checklist

- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md).
- [ ] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security. vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [ ] I have added tests that prove my fix is effective or that my feature
      works.
- [x] I have added or changed [the documentation](docs/docs).

### CODE REVIEW

File: docs/docs/guides/sign-in-with-github-google-facebook-linkedin.mdx [hunk 1]
```diff
@@ -98,8 +98,58 @@ selfservice:
...
+
+Your Jsonnet code snippet for the provider that only uses email address would look like this (you can place it in the same directory as your github one):
+
+```json title="contrib/quickstart/kratos/email-password/oidc.github.jsonnet"
+local claims = std.extVar('claims');
+
+{
+  identity: {
+    traits: {
+      email: claims.email
```
Comment (aeneasr) [inline docs/docs/guides/sign-in-with-github-google-facebook-linkedin.mdx:117]: It probably makes sense to copy this section from Github: https://github.com/ory/kratos/pull/424/files#diff-5beaed0e278de0cfec10f48986724b53R63-R76

File: docs/docs/guides/sign-in-with-github-google-facebook-linkedin.mdx [hunk 2]
```diff
@@ -98,8 +98,58 @@ selfservice:
...
+      enabled: true
+      config:
+        providers:
+          - id: github # this is `<provider-id>` in the Authorization callback URL. DO NOT CHANGE IT ONCE SET!
+            provider: github
+            client_id: .... # Replace this with the OAuth2 Client ID provided by GitHub
+            client_secret: .... # Replace this with the OAuth2 Client Secret provided by GitHub
+            mapper_url: file:///etc/config/kratos/oidc.github.jsonnet
+            scope:
+              - user:email
```
Comment (aeneasr) [inline docs/docs/guides/sign-in-with-github-google-facebook-linkedin.mdx:139]:
```suggestion
          # - id: github ...
```

File: docs/docs/guides/sign-in-with-github-google-facebook-linkedin.mdx [hunk 3]
```diff
@@ -3,15 +3,15 @@ id: sign-in-with-github-google-facebook-linkedin
...
+In this document we will take a look at setting up "Sign in with GitHub/Google" using
 ORY Kratos.
 
 Run the [Quickstart](../quickstart.mdx) with Docker Compose:
 
 ```shell script
 # This assumes that you have ORY Kratos checked out locally and
 # that your current directory ("pwd") is the folder where ORY Kratos
-# is.
+# is. Note that it is only tested against `v0.3.0-alpha.1` tag.
```
Comment (aeneasr) [inline docs/docs/guides/sign-in-with-github-google-facebook-linkedin.mdx:14]: This should work in future versions as well so we don't need to add this here :)

```suggestion
# is.
```

File: docs/docs/guides/sign-in-with-github-google-facebook-linkedin.mdx (hunk 1 above)
Comment (anuveyatsu) [reply docs/docs/guides/sign-in-with-github-google-facebook-linkedin.mdx:117]: @aeneasr do you the comment about verified email address? Would be relevant for google as well?

Comment (aeneasr) [reply docs/docs/guides/sign-in-with-github-google-facebook-linkedin.mdx:117]: Yes it is also relevant for Google!

**Comment by aeneasr** [comment]
Thank you, awesome!

**Pull request merged**